src/db.rs

```diff
@@ -2813,6 +2806,30 @@ impl<I: DBInner> DBCommon<MultiThreaded, I> {
         Ok(())
     }
 
+    #[doc = include_str!("db_create_column_family_with_import.md")]
+    pub fn create_column_family_with_import<N: AsRef<str>>(
+        &self,
+        options: &Options,
+        name: N,
+        import_options: &ImportColumnFamilyOptions,
+        metadata: &ExportImportFilesMetaData,
+    ) -> Result<(), Error> {
+        let inner = unsafe {
```

Comment:
same as my other comment, we would want something like:
```
        let name_ref = name.as_ref();
        let c_name = CString::new(name_ref).map_err(|err| {
            Error::new(format!(
                "Failed to convert column family name to CString when importing cf: {err}"
            ))
        })?;
        let inner = unsafe {
            ffi_try!(ffi::rocksdb_create_column_family_with_import(
                self.inner.inner(),
                options.inner,
                c_name.as_ptr(),
                import_options.inner,
                metadata.inner
            ))
        };
        self.cfs.cfs.write().insert(
            name_ref.to_owned(),
            Arc::new(UnboundColumnFamily { inner }),
        );
        Ok(())
```

Reply:
Fixed, thank you!